doc/apiref.md

```diff
@@ -80,7 +80,7 @@ master.
     stage_env_site_pp_path = get_site_pp_path(master, :env => 'staging')
     ```
 
-### `create_site_pp`
+### `create_site_pp(host, manifest)`
```

Comment:
The options hash is only documented in the examples. It would be nice to have a full description of what options are available.